Cargo.toml

```diff
@@ -1,8 +1,13 @@
 [package]
-name = "tremor"
+name = "librespot-tremor"
 version = "0.1.0"
-authors = ["Paul Lietar <paul@lietar.net>"]
+authors = ["Librespot Org"]
+build = "build.rs"
 
 [dependencies]
 libc = "*"
-tremor-sys = { path = "tremor-sys" }
+ogg-sys = "*"
+
+[build-dependencies]
+cc = "1.0.47"
```

Comment:
Can you explain these versions? What do they mean? Is this cross platform.

Reply:
I did not touch the Cargo.toml, it's all from the rebase I made. The current version on crates.io has probably the same Cargo.toml.

Reply:
I see. Sorry for the noise. Your changes look OK to me then.